release-process.md

```diff
@@ -0,0 +1,5 @@
+## Release Process
+1. Set the version number in ``pyproject.toml`` and ``cookiecutter/__init__.py`` files, and commit the changes.
```

Comment:
Blank line missed after header. Will fail on generating in rd

Reply:
the missing blank lines has been added